httpobs/database/schema.sql

```diff
@@ -24,6 +24,7 @@ CREATE TABLE IF NOT EXISTS scans (
   tests_quantity                      SMALLINT   NOT NULL,
   grade                               VARCHAR(2) NULL,
   score                               SMALLINT   NULL,
+  likelihood_indicator                VARCHAR(8) NULL,
```

Comment:
Also, could you just make this plain VARCHAR?  It's fine from a database perspective and it means I won't have to alter the database later should we come up with a wordier label.